custom_components/virtual_step_dimmer/config_flow.py

```
@@ -0,0 +1,82 @@
+
+    async def async_step_brightness(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the brightness steps step."""
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            try:
+                steps_str = user_input[CONF_BRIGHTNESS_STEPS]
+                steps = [int(s.strip()) for s in steps_str.split(",")]
+                if not steps or any(s <= 0 for s in steps):
+                    raise ValueError("Steps must be positive integers.")
```

Ruff (0.14.3)

Abstract `raise` to an inner function

(TRY301)